pages/common/distccd.md

```diff
@@ -0,0 +1,24 @@
+# distccd
+
+> Server daemon for the distcc distributed compiler.
+> More information: <https://distcc.github.io>.
+
+- Start a daemon with default settings:
+
+`distccd --daemon`
+
+- Start a daemon, accepting connections from private network IPv4 ranges:
```

Comment:
```suggestion
- Start a daemon, accepting connections from private IPv4 ranges:
```

Reply:
I'm not entirely sure. Here's my [source](https://wiki.archlinux.org/title/Distcc#Volunteers):

> At a minimum, add the --allow-private switch which covers a number of ipv4 private network ranges.

Perhaps that would be the preferred form?

```suggestion
- Start a daemon, accepting connections from IPv4 private network ranges:
```